api/server/api_0.cc

```diff
@@ -806,6 +812,14 @@ bool Api0::Convert(const MessageV0_NicUpdateReq &nic_update_message,
     } else {
         nic_update_model->has_ip_anti_spoof = false;
     }
+    // packet trace
+    if (nic_update_message.has_packet_trace()) {
+        nic_update_model->packet_trace = nic_update_message.packet_trace();
+        nic_update_model->has_packet_trace = true;
+    } else {
+        nic_update_model->packet_trace = app::config.packet_trace;
```

Comment:
I think you can remove `nic_update_model->packet_trace = app::config.packet_trace`